sdk/react-native/test/unit/PlatformEncryptionTools.test.ts

```diff
@@ -3,10 +3,12 @@
 import sinon from 'sinon'
 import { expect } from 'chai'
 import { KeysService } from '@affinidi/common'
+import { hmacSha256Verify } from 'eccrypto-js'
```

Comment:
@YuriiSkrypnyk @DenisPopov15 Have we check that eccrypto-js works well on android and ios ?

Reply:
It was already used by the each SDK in `PlatformEncryptionTools.ts` so it should be safe.